src/components/layout/SearchBarFooter/SearchBarFooter.tsx

```diff
@@ -1,5 +1,6 @@
 import './styles.css';
-import AntCard from '../../cards/AntCard/AntCard';
+import { AntCard } from '../../cards';
+import { FeaturedDomains } from '../';
```

Comment:
`FeaturedDomains` should be on the `HomePage` and not the `SearchBarFooter`- it doesn't have any relation to the search bar results/errors